### DISCUSSION TITLE
Microsoft Security Essentials indicates "Suspicious Behavior" with the fheroes2 version 1.0.5 "fheroes2_windows_x64_SDL2.zip" file

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#7310] [General]
Microsoft Security Essentials indicates "Suspicious Behavior" with the fheroes2 version 1.0.5 "fheroes2_windows_x64_SDL2.zip" file.

![Version 1 0 5 fheroes2_windows_x64_SDL2 zip](https://github.com/ihhub/fheroes2/assets/43583994/43e89b34-a7bc-4937-aad8-6ace3de13bd4)

**Comment by LeHerosInconnu** [comment]
Second attempt after system cleaning.

![Version 1 0 5 fheroes2_windows_x64_SDL2 zip 02](https://github.com/ihhub/fheroes2/assets/43583994/56290ad4-782d-4246-8b2a-0eed9681818a)

**Comment by LeHerosInconnu** [comment]
Third attempt.

![Version 1 0 5 fheroes2_windows_x64_SDL2 zip 03](https://github.com/ihhub/fheroes2/assets/43583994/14046792-20c5-442c-8623-922cd99537bc)

**Comment by LeHerosInconnu** [comment]
Trying to download the installer.

![Version 1 0 5 installer 03](https://github.com/ihhub/fheroes2/assets/43583994/1da8415c-19cc-497b-b6d2-b88f0a604c3b)

**Comment by LeHerosInconnu** [comment]
The problem no longer appears with version 1.0.5 7020.

**Reply by oleg-derevenetz** [reply]
Hmm, Microsoft Defender doesn't report anything for x64 installer as well as for ZIP archive from [here](https://github.com/ihhub/fheroes2/releases/tag/1.0.5) on my Windows 10 PC.

**Comment by LeHerosInconnu** [comment]
This is no longer relevant.
I close the discussion.
